fofix/core/GameEngine.py

```diff
@@ -358,18 +358,24 @@ def __init__(self, config = None):
             log.warn("No stages\ folder found, forcing None setting for Animated Stage.")
             self.config.set("game","animated_stage_folder", "None") #MFH: force "None" when Stages folder can't be found
 
-
-
         try:
+            self.theme = None
             fp, pathname, description = imp.find_module("CustomTheme",[themepath])
-            theme = imp.load_module("CustomTheme", fp, pathname, description)
-            self.theme = theme.CustomTheme(themepath, themename)
+            try:
+                theme = imp.load_module("CustomTheme", fp, pathname, description)
+                self.theme = theme.CustomTheme(themepath, themename)
+            except ImportError as e:
+                # Unable to load module; log it, but continue with default Theme.
+                log.error("Failed to load CustomTheme.py from %s" % pathname)
+                #self.theme = Theme(themepath, themename)
         except ImportError:
-            self.theme = Theme(themepath, themename)
+            # Not found; but that's OK. We'll use the default Theme class.
+            pass #self.theme = Theme(themepath, themename)
 
+        if not self.theme:
```

Comment:
Try this instead: `if self.theme is None`.

Reply:
Ok, I switched to "is None".

I recommend this go into CONTRIBUTING, as an argument can be made either way. Since this is Python, you can put anything into a variable at any time, and "if x" is a way of saying "I don't care what you put in x, I only care if I can use it". This is the preferred pattern in setup code, as it makes the code more tolerant of strange inputs.

On the other hand, "if x is None" has two benefits:  It's slightly faster, and if we get any developers who are amateurs, we want them to see and copy the "is None" pattern.

So I recommend putting a note in CONTRIBUTING along with your favorite reason.

Reply:
How to code in Python will not be in contributing guidelines.

Here, the variable is either `None` or a theme. Then, it has to be compared with `None` in this case.